src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PreReleaseServiceTests.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services;
+using Microsoft.Extensions.Options;
+using Xunit;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using System.Globalization;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+{
+    public class PreReleaseServiceTests
+    {
+        private static readonly DateTime DefaultScheduledPublishDate =
```

Comment:
Aha, fair enough. Will change! 
I often choose whatever constructor or parse method accepts an ISO string because it's language agnostic - but that's just a hangover from dealing with Javascript's native `Date` object which indexes *some* fields from 0 and others from 1... amongst other madness. Should of course have checked EES' conventions!